src/library/cgiparam/cgiparam.cpp

```diff
@@ -248,27 +248,24 @@ bool TCgiParameters::Has(const std::string_view name, const std::string_view val
 }
 
 TQuickCgiParam::TQuickCgiParam(const std::string_view cgiParamStr) {
-    UnescapeBuf.reserve(CgiUnescapeBufLen(cgiParamStr.size()));
-    char* buf = UnescapeBuf.data();
+    const size_t buf_length = CgiUnescapeBufLen(cgiParamStr.size());
```

Comment:
Неправильный стайгайд:
```suggestion
    const size_t bufLength = CgiUnescapeBufLen(cgiParamStr.size());
```